CinemaTicketSalesSystem/CinemaTicketSalesSystem/Views/Shared/MasterViewPartial.cshtml

```diff
@@ -0,0 +1,60 @@
+﻿@using Microsoft.AspNet.Identity
+@if (Request.IsAuthenticated)
+{
+
+    @Html.AntiForgeryToken()
+    if (User.IsInRole("Admin"))
```

Comment:
You can create the list of links for different roles and make a foreach for them instead of dublicating code